### DISCUSSION TITLE
None of my Links Display

### DISCUSSION THREAD

**Discussion by LynetteCullens** [l1xnan/obsidian-better-export-pdf#260] [Q&A]
[Obsidian Export](https://github.com/user-attachments/files/16141895/Glossary.pdf)
[Plugin Export](https://github.com/user-attachments/files/16141906/Glossary.pdf)
I don't even know how to diagnose this.

**Comment by l1xnan** [comment]
Can you provide a sanitized Markdown note? I need it to test and identify issues.

**Reply by LynetteCullens** [reply]
While building a sanitized version, I see that the links rendered fine when the reference is to the current page.

[Tester.md](https://github.com/user-attachments/files/16142739/Tester.md)

Looks like I have to write a script that embeds all links, and appends the block reference based on the folder location. I have a headache.
[I've improved it to this so far.](https://github.com/user-attachments/files/16143086/Glossary.pdf)

Applying the css from #259, I can get [this](https://github.com/user-attachments/files/16143150/Glossary.pdf)
